index.js

```diff
@@ -155,6 +164,40 @@ Cabal.prototype.publish = function (message, opts, cb) {
   })
 }
 
+/**
+ * Publish a message to your feed, encrypted to specific recipient's key.
+ * @param {String} text - The textual message to publish.
+ * @param {String|Buffer[32]) recipientKey - A recipient's public key to encrypt the message to.
+ * @param {function} cb - When the message has been successfully written.
+ */
+Cabal.prototype.publishPrivateMessage = function (text, recipientKey, cb) {
+  if (!cb) cb = noop
+  if (typeof text !== 'string') return process.nextTick(cb, new Error('text must be a string'))
+  if (!isHypercoreKey(recipientKey)) return process.nextTick(cb, new Error('recipientKey must be a 32-byte hypercore key'))
+
+  if (typeof recipientKey === 'string') recipientKey = Buffer.from(recipientKey, 'hex')
+
+  this.feed(function (feed) {
+    const message = {
+      type: 'private/text',
+      content: {
+        recipients: [recipientKey.toString('hex')],
```

Comment:
This is used in the private messages view so the sender knows who they sent their original message to. It could also be helpful if we add multiple participants later, so everyone knows who's in the convo.